content/docs/instrumenting/exporters.md

```diff
@@ -15,9 +15,15 @@ metrics directly (for example, HAProxy or Linux system stats).
 Some of these exporters are maintained as part of the official [Prometheus GitHub organization](https://github.com/prometheus),
 those are marked as *official*, others are externally contributed and maintained.
 
-We encourage the creation of more exporters but cannot
-vet all of them for [best practices](https://prometheus.io/docs/instrumenting/writing_exporters/). Commonly, those exporters are
-hosted outside of the Prometheus GitHub organization.
+We encourage the creation of more exporters but cannot vet all of them for
```

Comment:
Btw., https://en.oxforddictionaries.com/usage/cannot-or-can-not: "Both cannot and can not are acceptable spellings, but the first is much more usual."